### DISCUSSION TITLE
GraphQL Query - Can't get related asset if the field handle has been customized.

### DISCUSSION THREAD

**Discussion by thisanimus** [craftcms/cms#15623] [Q&A] [answered]
I have customized a field handle (`resourceThumbnail`->`thumbnail`) to make things more consistent for us in the frontend.  I have the following query:
```
query MyQuery {
  assets(volume: "resourceAssets") {
    id
    ... on resourceAssets_Asset {
      thumbnail {
        id
      }
    }
  }
}
```
where `thumbnail` is a related asset.  I get an empty array for `thumbnail`.

If I remove the handle customization and query the original handle, I get the thumbnail object.
```
query MyQuery {
  assets(volume: "resourceAssets") {
    id
    ... on resourceAssets_Asset {
      resourceThumbnail {
        id
      }
    }
  }
}
```

Am I missing something here?  I think the first query should be returning the related asset details right?

**Comment by brandonkelly** [comment]
Just tested this locally and accessing the relations via the overridden handle is working for me.

The `resourceThumbnail` is on your volume’s field layout, correct?

**Comment by thisanimus** [comment, marked as answer]
Apparently something about the GraphQL schema got stuck.  I re-applied the schema and that fixed it.

**Reply by brandonkelly** [reply]
Sweet!
